BDA_FS19_Projektdokumentation.tex

```diff
@@ -277,6 +277,76 @@ \subsection{Grobplan}
 
 \newpage
 
+\section{Tools}
+\label{sec:Tools}
+
+\subsection{Versions Kontrolle}
+\begin{table}[h!]
+	\begin{tabular}{p{0.5\textwidth} p{0.5\textwidth}}
+		\hline
+		\textbf{Tool} & \textbf{Version} \\
+		\hline
+		GitKraken & 4.2.2 \\
+		\hline
+		git & 2.21.0 \\
+		\hline
+		GitHub & GitHub.com \\
+		\hline
+	\end{tabular}
+	\caption{Verwendete Versionen der Versionierungssysteme}
+\end{table}
+
+\subsection{Dokumentation}
+\begin{table}[h!]
+	\begin{tabular}{p{0.5\textwidth} p{0.5\textwidth}}
+		\hline
+		\textbf{Tool} & \textbf{Version} \\
+		\hline
+		TeX Live & 2018 \\
+		\hline
+		MiKTex & 2.9.6637 \\
+		\hline
+		Mendeley Desktop & 1.19.2 \\
+		\hline
+		MS Excel & 16.0.11231.20164 \\
+		\hline
+		OneDrive & - \\
```

Comment:
Either the Client:
19.012.0121.0011
https://support.office.com/de-de/article/neuer-onedrive-synchronisierungsclient-%e2%80%93-anmerkungen-zu-dieser-version-845dcf18-f921-435e-bf28-4e24b95e5fc0?ui=de-DE&rs=de-DE&ad=DE

Or just OneDrive Online